data/dummyTripsData.js

```diff
@@ -0,0 +1,41 @@
+const trips = [
```

Comment:
This file I honestly feel may need to most changes as I see what sort of things I need to be able to flexibly test.